.github/workflows/test.yaml

```diff
@@ -44,7 +44,7 @@ jobs:
           go-version-file: go.mod
 
       - name: Run unit tests
-        run: make all
+        run: make test
```

Comment:
I think we do want to run the `check-unused` Makefile target -- otherwise, we can't guarantee that engineers miss to vendor dependencies correctly.